Makefile

```diff
@@ -242,6 +242,18 @@ test-e2e:
 test-e2e-skip-upgrade:
 	@VERSION=$(VERSION) OSMOSIS_E2E_SKIP_UPGRADE=True go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E)
 
+lo-keys:
+	. tests/localosmosis/keys.sh
+
+lo-build:
+	@docker build -t local:osmosis --build-arg BASE_IMG_TAG=debug -f tests/localosmosis/Dockerfile .
```

Comment:
`--build-arg BASE_IMG_TAG=debug` can be removed as it's not used.

If you like the idea of adding the `build/context` option in the `docker-compose` as suggested in a previous command, the command could be made consistent to the others replacing it with:

 `@docker-compose -f tests/localosmosis/docker-compose.yml build`